openapi/components/schemas/Order.yaml

```
@@ -0,0 +1,197 @@
+      such as suspending the service.
+    type: string
+    readOnly: true
+    enum:
+      - draft
+      - unpaid
+      - past-due
+      - abandoned
+      - paid
+      - voided
+      - refunded
+      - disputed
+      - partially-refunded
+      - partially-paid
+  items:
+    type: array
+    description: |-
+      Items of the order.
+
+      Note: Currently, this collection should contain no more than 1 element.
+    items:
+      type: object
+      description: Order entry details.
+      discriminator:
+        propertyName: type
+        mapping:
+          subscription: ./Subscription.yaml
+          one-time-sale: ./OneTimeSale.yaml
+      anyOf:
+        - $ref: ./Subscription.yaml
+        - $ref: ./OneTimeSale.yaml
+  invoiceIssueTimeShift:
+    description: |-
+      Invoice issue time shift interval in ISO-8601 format.
+
+      This field adjusts the issue date and time of renewal invoices for the order.
+
+      This interval value supports the `-` symbol.
+    type:
+      - 'string'
+      - 'null'
+    example: +P00Y00M00DT00H00M00S
+  invoiceDueTimeShift:
+    description: |-
+      Invoice due time shift interval in ISO-8601 format.
+
+      This field adjusts when the payment is due for the order.
+      If autopay is configured, payment is collected automatically on this date.
+
+      This interval value supports the `+` symbol.
+    type:
+      - 'string'
+      - 'null'
+    example: +P00Y00M00DT00H00M00S
+  paymentInstrumentId:
+    type:
+      - 'string'
+      - 'null'
+    description: |-
+      ID of the payment instrument to use for autopay.
+      If this value is not provided, or if the payment instrument is inactive,
+      the customer's default payment instrument is used.
+    maxLength: 50
+    example: inst_0YVB8KPKNXCBR9EDX7JHSED75N
+  poNumber:
+    description: Purchase order number that is displayed on the issued invoices.
+    type:
+      - 'string'
+      - 'null'
+    example: PO123456
+  notes:
+    description: Notes for the customer that are displayed on the order invoice.
+    type: string
+  billingAddress:
+    description: Billing address of the order.
+    oneOf:
+      - $ref: ./ContactObject.yaml
+      - type: 'null'
+  deliveryAddress:
+    description: Delivery address of the order.
+    oneOf:
+      - $ref: ./ContactObject.yaml
+      - type: 'null'
+  delinquencyPeriod:
+    type:
+      - 'string'
+      - 'null'
+    description: |-
+      Length of time, in ISO-8601 durations format,
+      which is added to the due time of the order when setting the delinquency time for all related invoices.
+      When the delinquency time of an invoice is reached, the order is automatically canceled.
+      The current invoice is abandoned along with order invoices that have the `unpaid` or `past-due` status.
+
+      If the provided value is `null`, the order does not change state.
+    example: +P00Y00M07DT00H00M00S
+    default: null
+  richBillingPortalId:
+    type: string
```

```suggestion
    type:
      - string
      - 'null'
```
In creating an order it shows this can be `null`. So this should be defined like that, right?

---

Yes, it's `nullable`, updated, thank you